### ISSUE TITLE
new site with "g00" behavior

### ISSUE THREAD

**Issue by Procyon-b** [gorhill/uBO-Extra#111]
I've encountered a new site with the same behavior as msn: page load interrupted -> reloading using g00 urls.
I've fixed it by disallowing one script, but I don't know if it breaks others things on this site. I don't know if it should be fixed with a filter in one of the standard filters lists, or if uBO extra is the solution.

https://www.narcity.com/ca/qc/montreal/lifestyle/harry-potter-et-leducation-sexuelle-une-parodie-de-sarah-jeanne-labrosse-et-pier-luc-funk-a-mourir-de-rire

**Note about the msn example:**
Until recently I was using an old version of uBO, more specificaly, a clone/fork (until 2018/11/13), and discovered the "g00" behavior only after installing the lastest uBO (version 1.17). I have downloaded and installed older version of uBO, and discovered that msn uses "g00" only with uBO version >=1.16.
v1.15.24 and lower doesn't trigger the behavior (and so, no backdoor used to load ads).

Do you want me to file a new issue in uBO (or here) with all the infos to reproduce the test?

**Comment by mapx-** [comment]
Could you test this filter
`narcity.com##+js(abort-on-property-read.js, I10C)`

**Comment by okiehsch** [comment]
Works on my end.

**Comment by okiehsch** [comment]
> Note about the msn example:
Until recently I was using an old version of uBO, more specificaly, a clone/fork (until 2018/11/13), and discovered the "g00" behavior only after installing the lastest uBO (version 1.17). I have downloaded and installed older version of uBO, and discovered that msn uses "g00" only with uBO version >=1.16.
v1.15.24 and lower doesn't trigger the behavior (and so, no backdoor used to load ads).


I am not sure there is something gorhill can do about that, but I reopened because I missed that part of the issue.

**Comment by Procyon-b** [comment]
FYI I'm currently using this filter:
https://www.narcity.com/assets/

**Comment by okiehsch** [comment]
If you update your `uBO-filters` list you should be able to remove that filter.

**Comment by Procyon-b** [comment]
It's fixed with uBO-filters reloaded.
Is that somehing you've fixed just now?

**Comment by okiehsch** [comment]
![image](https://user-images.githubusercontent.com/16838044/49970363-49b3d900-ff2b-11e8-8788-bc8ccbe35bd2.png)

**Comment by Procyon-b** [comment]
That was fast  :)

**Comment by gorhill** [comment]
> discovered that msn uses "g00" only with uBO version >=1.16. v1.15.24 and lower doesn't trigger the behavior

It can be quite time consuming to investigate IL stuff, so without investigating I can only speculate. The change that stand out in 1.16.0 is that uBO started to use user stylesheet in [1.16.0](https://github.com/gorhill/uBlock/releases/tag/1.16.0), which suggests that IL was using uBO's injected stylesheet in the DOM to decide whether or not it acivated itself.

**Comment by terrorist96** [comment]
I confirm that I see ads at that link.
![image](https://user-images.githubusercontent.com/12802056/53304002-bb1eda80-383e-11e9-845e-f33745bcbabe.png)

**Comment by terrorist96** [comment]
@okiehsch thanks, fixed on my end. Can this be closed?
